Notes mode (#54)
ankitects/anki-manual: browsing.md

The Browse window allows you to search through your cards and notes, and edit
them. It is opened by clicking **Browse** in the main window, or by pressing
**b**. It is comprised of three sections: the *sidebar* on the
left, the *card/note table* on the top right, and the *current note* on the bottom
right. By positioning the mouse between two sections, it is possible to click
and drag to expand one section and shrink the other.

## Table Modes

Anki 2.1.43+ offers two modes to show either cards or notes in the data table.
You can change the current mode by clicking the switch in the top left corner

> right of the sidebar or pressing <kbd>Alt</kbd>+<kbd>T</kbd>. The switch also

Author: Wouldn't that mean that it's inside the sidebar? Maybe "at the top between sidebar and searchbar"?

> right of the sidebar or pressing <kbd>Alt</kbd>+<kbd>T</kbd>. The switch also

Reviewer: "At the top, to the left of the search area"?

> right of the sidebar or pressing <kbd>Alt</kbd>+<kbd>T</kbd>. The switch also

Reviewer: I think people will hopefully be able to find it no matter what we choose :-)

according search expression:
`deck:Urdu (is:due or tag:important)`.

### Selection Tool

The Selection tool allows for selecting multiple items at the same time by <kbd>Ctrl</kbd>-
and <kbd>Shift</kbd>-clicking. It also enables drag and drop to reorder decks and
tags.

Here is an example: Say you have the tags `Math`, `Calculus` and `Algebra`.
Click on the `Calculus` tag, then <kbd>Ctrl</kbd>-click on the `Algebra` tag. Now both
tags are selected, click and drag any of the two onto the `Math` tag to make them
both children of this tag. Behind the curtains, Anki has renamed the two tags to
`Math::Calculus` and `Math::Algebra` respectively and updated your notes accordingly.

> Another use case for selecting multiple items is searching: If you rightclick on

Reviewer: ```suggestion
Another use case for selecting multiple items is searching: If you right-click on
```

a selection of items, you can choose **Search &gt; All/Any Selected Items**. This
can be combined with keyboard modifiers as described in [Search Tool](#search-tool)
to append the resulting search to the current search.

### Saved Searches

If you regularly search for the same thing,

> you can save the current search by rightclicking the topmost item in the sidebar,

Reviewer: ```suggestion
you can save the current search by right-clicking the topmost item in the sidebar,
```

choosing “Save Current Search” and typing in a name.

### Editing Items

> You can delete or rename tags, decks and saved searches directly from the sidebar

Reviewer: ```suggestion
You can delete or rename tags, decks and saved searches directly from the sidebar,
```

> through the context menu or using the according keys (<kbd>Del</kbd> and <kbd>F2</kbd>

Reviewer: ```suggestion
via the right-click menu, or by using a shortcut key (<kbd>Del</kbd> and <kbd>F2</kbd>
```

When you click on a row, the corresponding note will be shown in the bottom section.

### Rows

If you drag the mouse or hold <kbd>Ctrl</kbd> or <kbd>Command</kbd> to select multiple
rows, the editor will be temporarily hidden. Various operations (such as
changing the deck) can operate on multiple cards or notes at once, independent
of the active mode. Therefore in Cards mode, a note is considered to be selected
if any of its cards is selected, and in Notes mode, a card is considered to be selected
if its note is selected.

Other operations (like showing card info) only operate on a single, the so-called
*current*, card or note, which is usually the one that was last left- or rightclicked.
In Cards Mode, again, the current note is the note of the current card and in Notes
mode, the current card is the first card of the current note.

The background color will change depending on the card and note.
In Cards mode, Anki will change the color of the row depending on the state of
the card. The first match will be used: 1. if flagged, use the flag color; 2. if
suspended, yellow; 3. if marked, purple. In Notes mode, color is only applied to

> marked notes.  

Reviewer: Maybe this would be a little easier to read as a separate list, instead of contained in the paragraph?

the last time changes were made to the _card_ (e.g., when you reviewed
the card and the review history and interval were updated).

When you click on a row, the corresponding note will be shown in the bottom section.
If you drag the mouse or hold <kbd>Ctrl</kbd> or <kbd>Command</kbd> to select multiple
rows, the editor will be temporarily hidden. Various operations (such as
changing the deck) can operate on multiple cards or notes at once, independent
of the active mode. Therefore in Cards Mode, a note is considered to be selected
if any of its cards is selected and in Notes Mode, a card is considered to be selected
if its note is selected. If an operation only works for a single card (like
showing card info) and Notes Mode is active, it's performed on the current note's
card with index 1.

The background color will change depending on the card and note.

> In Cards Mode, a row is assigned the first applicable of the following colors:

Reviewer: ```suggestion
In Cards mode, a row is assigned the first applicable of the following colors:
```

Maybe a lowercase mode would be better in each of the references? Don't have strong feelings about it though.

The color of its flag if any; a shade of yellow if the card is suspended; a
shade of purple if its note is marked. In Notes Mode, only the last criterion

> applies.  

Reviewer: This is a bit technical-sounding. How about something like this?

In Cards mode, Anki will change the color of the row depending on the state of the card. The first match will be used:

1. if flagged, use the flag color
2. if suspended, yellow
3. if marked, purple

In Notes mode, color is only applied to marked notes.

### Cards

The following actions operate on the selected cards. They are also available through
a context menu when a selected row is rightclicked in Cards mode. In Notes mode,
they can be found in a submenu of the context menu.

| Name | Action |
|-|-|
| Change Deck | Move all selected cards to a different deck. Cards of the same note can be in different decks. If you want to make sure this doesn't happen, you can use **Edit &gt; Select Notes** first or perform the deck change in Notes mode. |
| Set Due Date | Move cards to the end of the new card queue, or reschedule them as a review card on a given date. The second option is useful if you have imported already-learnt material, and you want to start it off with higher initial intervals. For example, choosing 60 and 90 will give all the imported cards an initial interval of 2 to 3 months.The card’s revision history is not cleared when rescheduling: rescheduling changes the current state of a card, but not its history. If you want to hide the history, you will need to export your notes as a text file, delete the notes, and then import the text file again, creating new notes. |
| Forget |  |

> | Reposition | Change the order new cards will appear in. You can find out the existing positions by enabling the _due_ column, as described in the [table](#cardnote-table) section above. If you run the reposition command when multiple cards are selected, it will apply increasing numbers to each card in turn. By default the number increases by one for each card, but this can be adjusted by changing the "step" setting. The **Shift position of existing cards** option allows you to insert cards between currently existing ones, pushing the currently existing ones apart. For instance, if you have five cards and you want to move 3, 4, and 5 between 1 and 2, selecting this setting would cause the cards to end up in the order 1, 3, 4, 5, 2. By contrast, if you turn this option off, 1 and 2 will get the same position number (and it will thus be unpredictable which of the cards with the same number comes up first). |

Reviewer: ```suggestion
| Reposition | Change the order new cards will appear in. You can find out the existing positions by enabling the _due_ column, as described in the [table](#cardnote-table) section above. If you run the reposition command when multiple cards are selected, it will apply increasing numbers to each card in turn. By default the number increases by one for each card, but this can be adjusted by changing the "step" setting. The **Shift position of existing cards** option allows you to insert cards between currently existing ones, pushing the currently existing ones apart. For instance, if you have five cards and you want to move 3, 4, and 5 between 1 and 2, selecting this setting would cause the cards to end up in the order 1, 3, 4, 5, 2. By contrast, if you turn this option off, 1 and 2 will get the same position number (and it will thus be unpredictable which of the cards with the same number comes up first). Please note that when enabled, any card with a higher position will be modified, and all of those changed cards will need to be sent the next time you sync.  ||
```

Reviewer: Hmm, diff didn't work well there - I added:

Please note that when enabled, any card with a higher position will be modified, and all of those changed cards will need to be sent the next time you sync.  |

Author: One of the drawbacks of having to write ridiculously long lines...
Thanks for writing it out for me.

| Toggle Suspend | [Suspend](studying.md#editing-and-more) or unsuspend all selected cards, depending on whether the current card is suspended or not. |

> | Flag | Toggle the flags of all selected cards. Whether a flag is added or removed depends on whether the current card has this flag. |

Reviewer: ```suggestion
| Flag | Toggle the flags of all selected cards. Whether a flag is added or removed depends on whether the current card has the chosen flag. |
```
